README.md

```diff
@@ -66,6 +70,19 @@ Set `non-nil` if you use `elscreen`
 (global-set-key (kbd "C-x C-z") 'zoom-window-zoom)
 ```
 
+### zoom-window with [persp-mode](https://github.com/Bad-ptr/persp-mode.el)
+
+```lisp
+(require 'elscreen)
```

Comment:
These two lines are elscreen configuration. Please check and fix for persp-mode.